app/controllers/build_allocations_controller.rb

```diff
@@ -6,6 +6,7 @@ class BuildAllocationsController < PrisonsApplicationController
   before_action :set_referrer
 
   include Wicked::Wizard
+  include OffenderHelper
```

Comment:
I'm not sure about this. I've always thought of helpers for use by views only, what do you think? Happy to proceed if you feel strongly that helpers are meant to be shared between views and controllers.